pkg/generator/gardenlinux/files/kubelet_cgroup_driver.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+function get_fs_of_directory {
+    [ -z "$1" -o ! -d "$1" ] && return
+    echo -n "$(stat -c %T -f "$1")"
+}
+
+function check_current_cgroup {
```

Comment:
Yes, I moved these functions which were present in three different scripts into a separate file `g_functions.sh` which is then sourced by the other scripts. Makes total sense to avoid this kind of code duplication.